src/MCPClient-base.Dockerfile

```diff
@@ -62,6 +62,7 @@ RUN set -ex \
 		ufraw \
 		unrar-free \
 		uuid \
+		unzip \
```

Comment:
Is this needed by 7z?

Reply:
It's needed by `atool` when it tries to decompress during AIP verification.

Reply:
Oh I had no idea, thanks!